config/locales/controllers/en.yml

```diff
@@ -25,6 +25,10 @@ en:
         success:
           one: Successfully removed pseud %{pseuds} from this work.
           other: Successfully removed pseuds %{pseuds} from this work.
+  application:
+    access_denied:
```

Comment:
To avoid the duplicate key name, could this be `auth` or something similar? That way, we could put other authentication-related messages under here if we ever have those

Reply:
It would be nicer to change the inner two keys instead of the outer ones, because i18n-tasks will by default grab the controller and method name for the relative key. Would `application.access_denied.auth.logged_in` work?

Reply:
Hmm, in that case I think it's fine to be honest